GTG/gtk/preferences.py

```diff
@@ -165,35 +156,22 @@ def valid_check(self, widget):
         self.refresh_hour.modify_fg(Gtk.StateFlags.NORMAL, color)
         self.refresh_mins.modify_fg(Gtk.StateFlags.NORMAL, color)
 
-    def on_close(self, widget, data=None):
```

Comment:
The old way to do things in GTG was to initiate all windows by init and then just hide/reactivate them. This method hides the dialog so it can be reactivated (`show_all`)

This behaviour will be changed in the PR I'm working on